irsdb/filing/management/commands/enter_yearly_submissions.py

```diff
@@ -43,7 +42,13 @@ def handle(self, *args, **options):
             next(reader)
             count = 0
             for line in reader:
-                (return_id, filing_type, ein, tax_period, sub_date, taxpayer_name, return_type, dln, object_id) = line
+                try:
+                    (return_id, filing_type, ein, tax_period, sub_date, taxpayer_name, return_type, dln, object_id) = line
+                except ValueError as err:
+                    print("Error with line: {line}".format(line=line))
+                    if year == 2014:
+                        print('Did you fix the 2014 index file? See the README for instructions.')
+                    raise err
```

Comment:
I think this should just be `raise` to re-raise the same error from the same call stack; raising in this fashion will make it look like the error was triggered here.